examples/challenge_response.rs

```diff
@@ -0,0 +1,38 @@
+extern crate reqwest;
+extern crate veraison_apiclient;
+
+fn my_evidence_builder(
+    nonce: &Vec<u8>,
+    accept: &Vec<String>,
```

Comment:
Normally a reference to a `Vec` is written as `&[u8]` (for example), though I think there are a few cases where it's better to use `&Vec`, but can't tell if it applies here.

Reply:
thanks, I'll have a look at the documentation.

Reply:
As far as I remember, `&[T]` is essentially a superset of `&Vec<T>` (I think it's [this](https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-AsRef%3C%5BT%5D%3E) implementation behind it), but it really isn't that critical here

Reply:
ok, done in [464be85](https://github.com/veraison/rust-apiclient/pull/4/commits/464be85bf725585835599fcb0ab3d6699e15e806).  I still don't understand the advantage of one over the other, my learning curve is still in its infancy.